src/DependencyInjection/EzPlatformHttpCacheExtension.php

```diff
@@ -36,6 +37,15 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('services.yml');
         $loader->load('event.yml');
         $loader->load('view_cache.yml');
+
+        $purgeType = $container->getParameter('ezpublish.http_cache.purge_type');
+        if ('local' === $purgeType) {
```

Comment:
We had `http` which is gone, have `varnish` and this one. Was there anything else?

Reply:
`fastly`, but in other EE bundle